### ISSUE TITLE
TemplateRuntimeError: no test named 'equalto' on RHEL 7 control nodes

### ISSUE THREAD

**Issue by berndfinger** [linux-system-roles/sap-netweaver-preconfigure#42]
after fixing #33 .

**Comment by richm** [comment]
Right.  `equal`, `equalto`, and `==` are not available with jinja 2.7.  You must use `match`.

Instead of 
`selectattr('number', 'equalto', '2526952')`
use
`selectattr('number', 'match', '^2526952$')`
